src/muqsit/vanillagenerator/generator/VanillaGenerator.php

```diff
@@ -18,6 +18,7 @@ abstract class VanillaGenerator extends Generator{
 
 	protected const WORLD_DEPTH = 128;
 
+	// TODO Figure out a non hacky way to use this. We don't have access to the ChunkManager necessary to call it in the constructor
```

Comment:
you may remove this, there is no non-hacky method to access a non-public property

Reply:
Remove the method entirely, or just the comment- the method isn't called anymore.

Reply:
Yeah, you may as well remove all of this entirely: https://github.com/adams-pm-plugins/VanillaGenerator/blob/master/src/muqsit/vanillagenerator/generator/VanillaGenerator.php#L19-#L32